helpers/prompts.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+
+const { validateIdentifier, validateNotEmpty, validatePath, validateVersion } = require('./validators.js');
+
+let taoPath = '';
+
+module.exports = {
+    pciPath: {
+        type: 'input',
+        name: 'pciPath',
+        message: 'Where do we generate the PCI (path)?',
+        initial: '.',
+        validate: validatePath
+    },
+    typeIdentifier: {
+        type: 'input',
+        name: 'typeIdentifier',
+        message: "What's the PCI typeIdentifier?",
```

Comment:
```suggestion
        message: 'What\'s the PCI typeIdentifier?',
```

Reply:
Indeed, this is a way of strictly follow the code style. But it is also more slashy. This is one of the exceptions the code style accepts, and since the original code had it, I chose to keep it (Prettier was ok with it as well 😄)